kotekan/kotekan.cpp

```diff
@@ -124,6 +124,8 @@ kotekanMode* kotekan_mode = nullptr;
 bool running = false;
 std::mutex kotekan_state_lock;
 volatile std::sig_atomic_t sig_value = 0;
+extern enum ReturnCode status_code;
```

Comment:
You could have a function like `get_exit_code()` in `errors.h` to avoid having to `extern` the variable holding the value here.  That's a little cleaner than doing it this way.